WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -128,6 +128,38 @@ $venue-map-wrapper-height: 400px;
 
 .competitions-table {
 
+  p {
+    margin: 0;
+  }
+
+  tr {
+    &.past {
+      a:link,
+      a:visited {
+        color: #139;
+      }
+    }
+
+    &.not-past {
+      a:link,
+      a:visited {
+        color: #e64503;
```

Comment:
You've got 2 colors here, how about giving them names and putting them in inside of `variables.scss`?